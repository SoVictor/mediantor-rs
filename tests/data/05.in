5
1 -840167367
0
1 -627664644
1 142271917
0
