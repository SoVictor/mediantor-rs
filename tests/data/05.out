-840167367
-627664644
